src/components/audio-entry-details/_audio-entry-details.scss

```diff
@@ -55,8 +55,8 @@
 
     &.audio-entry-expanded {
       overflow: auto;
-      background: rgba(0, 0, 0, 0.7);
-      height: 100%;
+      background: linear-gradient(180deg, rgba(0, 0, 0, 0) 0%, rgba(0, 0, 0, 0.7) 100%);
```

Comment:
Don’t use hardcoded values , use the equivalent dynamic css var (otherwise light mode won’t be supported)